src/apps/testapps/testVertexGraph.c

```diff
@@ -236,6 +236,9 @@ TEST(removeVertexNode) {
 
     // Remove non-existent node
     node = malloc(sizeof(VertexNode));
+    node->next = NULL;
```

Comment:
Easier just to use `calloc` instead of `malloc` here?

Reply:
It would be easier; done.